pkgs/development/python-modules/openapi3/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, requests
+, pyyaml
+}:
+
+buildPythonPackage rec {
+  pname = "openapi3";
+  version = "1.8.2";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.8";
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-ohpJBXPYnKaa2ny+WFrbL8pJZCV/bzod9THxKBVFXSw=";
+  };
+
+  nativeBuildInputs = [
+    requests
+    pyyaml
+  ];
```

```suggestion
  ];
  
  nativeCheckinputs = [
    pytestCheckHook
  ];
  
  pythonImportsCheck = [
    "openapi3"
  ];
```
Could you please enable checkPhase?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages